postgresql/helpers.go

```diff
@@ -105,6 +105,7 @@ func sliceContainsStr(haystack []string, needle string) bool {
 // allowedPrivileges is the list of privileges allowed per object types in Postgres.
 // see: https://www.postgresql.org/docs/current/sql-grant.html
 var allowedPrivileges = map[string][]string{
+	"database": []string{"ALL", "ALL PRIVILEGES", "CREATE", "CONNECT", "TEMPORARY", "TEMP"},
```

Comment:
You can let them to be coherent with the others but note that there's a bug with `ALL` privileges #72 
(I would have let only `ALL` and not `ALL PRIVILEGES` to be coherent too but your choice)